code/modules/mob/living/simple_animal/friendly/dog.dm

```diff
@@ -420,13 +420,18 @@
 	unique_pet = TRUE
 	var/age = 0
 	var/record_age = 1
-	var/memory_saved = FALSE
 	var/saved_head //path
 
 /mob/living/simple_animal/pet/dog/corgi/Ian/Initialize(mapload)
 	. = ..()
-	//parent call must happen first to ensure IAN
-	//is not in nullspace when child puppies spawn
+	SSpersistent_data.register(src)
+
+/mob/living/simple_animal/pet/dog/corgi/Ian/death()
```

Comment:
Missing a param here `gibbed`

Reply:
Param is not used anywhere and its not halting dreamchecker